requirements.txt

```diff
@@ -1,7 +1,7 @@
 pyhive[hive_pure_sasl]~=0.7.0
 requests>=2.28.1
 
-pyodbc~=4.0.39 --no-binary pyodbc
+pyodbc~=5.1.0 --no-binary pyodbc
```

Comment:
While I agree with bumping pyodbc if we can, is this required for this PR? Or can we do it as a separate PR?

Reply:
@mikealfare  It's actually required for this PR

Reply:
Ah yes, I looked it up backwards.